### PR TITLE
refactor: show button under iframe for manual opening

### LINKED ISSUE #81: Open link manually button disappeared

### PR DESCRIPTION
**Pull request by pyphilia**
This button was buggy, so I changed for a regular button under the iframe. This is also better for accessibility.

close #81

### CODE REVIEW

File: src/items/LinkItem.tsx [hunk 1]
```diff
@@ -60,12 +46,12 @@ const LinkItem: FC<LinkItemProps> = ({
   saveButtonId,
   editCaption = false,
   showCaption = true,
-  loadingMessage = 'Link is Loading.',
-  openLinkMessage = 'Click here to open link manually.',
+  loadingMessage = 'Link is Loading...',
+  openLinkMessage = 'Click here to open the link manually',
   height: defaultHeight,
+  errorMessage = 'The link is malformed.',
```
Comment (juancarlosfarah) [inline src/items/LinkItem.tsx:52]: Are these messages hard-coded? Or can they be overridden when used? It's probably better if everything can be overridden.

File: src/items/LinkItem.tsx [hunk 2]
```diff
@@ -105,6 +95,10 @@ const LinkItem: FC<LinkItemProps> = ({
     // }
   };
 
+  const onClick = (): void => {
+    redirect(url, { openInNewTab: true });
+  };
+
   const component = (
     <React.Fragment>
```
Comment (juancarlosfarah) [inline src/items/LinkItem.tsx:103]: I would import { Fragment } from React. Looks nicer and we're already importing things.

**Review by juancarlosfarah — APPROVED** [review]
Approving with some minor comments / questions! 🪨

File: src/items/LinkItem.tsx (hunk 1 above)
Comment (pyphilia) [reply src/items/LinkItem.tsx:52]: These are default values, so yes they can be overridden!

**Review by juancarlosfarah — APPROVED** [review]
👍

**Pull request merged**